docker/Dockerfile

```diff
@@ -7,7 +7,7 @@ MAINTAINER andreas-bok-sociomantic <andreas.bok@sociomantic.com>
 # Set environment variables
 ENV MYSQL_USER=user \
     MYSQL_PASSWORD=pass \
-    MYSQL_ROOT_PASSWORD='' \
+    MYSQL_ROOT_PASSWORD=pass \
```

Comment:
I see what you mean. In the unittest it is the **MYSQL_USER** which is used, so setting **MYSQL_ROOT_PASSWORD** is only to comply with the policy of MySQL that root should have a password. 
I would rather keep as is, if it's ok with you. It is only exposed to the Travis CI, so the user of library wouldn't have to deal with this env variable anyway